docs/source/daml/intro/4_Transformations.rst

```diff
@@ -76,13 +76,14 @@ Choices in the Ledger Model
 
 In :doc:`1_Token` you learned about the high-level structure of a DAML ledger. With choices and the `exercise` function, you have the next important ingredient to understand the structure of the ledger and transactions.
 
-A *transaction* is a list of *actions*, and there are just three kinds of action: ``create``, ``exercise`` and ``fetch``. All actions are performed on a contract.
+A *transaction* is a list of *actions*, and there are just four kinds of action: ``create``, ``exercise``, ``fetch`` and ``key assertion``.
 
-- A ``create`` action contains the contract arguments and changes the contract's status from *non-existent* to *active*.
-- A ``fetch`` action  checks the existence and activeness of a contract.
-- An ``exercise`` action contains the choice arguments and a transaction called the *consequences*. Exercises come in two kinds called ``consuming`` and ``nonconsuming``. ``consuming`` is the default kind and changes the contract's status from *active* to *archived*.
+- A ``create`` action creates a new contract with the given arguments and sets its status to *active*.
+- A ``fetch`` action checks the existence and activeness of a contract.
+- An ``exercise`` action exercises a choice on a contract resulting in a transaction (list of sub-actions) called the *consequences*. Exercises come in two kinds called ``consuming`` and ``nonconsuming``. ``consuming`` is the default kind and changes the contract's status from *active* to *archived*.
+- A ``key assertion`` records the assertion that the given contract key (see :ref:`contract_keys`) is not assigned to any active contract on the ledger.
```

Comment:
Note: I used the word _active_, while https://docs.daml.com/concepts/ledger-model/ledger-structure.html used the word _unconsumed_.